MIGRATION.md

```diff
@@ -339,6 +340,14 @@
 
 ## From version 7.x to 8.0.0
 
+### Removal of `storiesOf`-API
+
+The `storiesOf` API has been removed in storybook 8.0.
```

Comment:
@ndelangen, one small item here, if you wouldn't mind capitalizing Storybook I'd appreciate it 🙏 !